<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HEXRD</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="central_widget_layout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="ImageTabWidget" name="image_tab_widget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menu_open">
     <property name="title">
      <string>&amp;Open</string>
     </property>
     <addaction name="action_open_images"/>
     <addaction name="action_open_config"/>
     <addaction name="action_open_materials"/>
     <addaction name="action_open_aps_imageseries"/>
     <addaction name="action_open_template_dialog"/>
    </widget>
    <widget class="QMenu" name="menu_save">
     <property name="title">
      <string>&amp;Save</string>
     </property>
     <addaction name="action_save_imageseries"/>
     <addaction name="action_save_config"/>
     <addaction name="action_save_materials"/>
    </widget>
    <widget class="QMenu" name="menu_export">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="action_export_polar_plot"/>
    </widget>
    <addaction name="menu_open"/>
    <addaction name="menu_save"/>
    <addaction name="menu_export"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_show_toolbar"/>
    <addaction name="action_show_live_updates"/>
    <addaction name="action_show_detector_borders"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>E&amp;dit</string>
    </property>
    <addaction name="action_edit_euler_angle_convention"/>
    <addaction name="action_edit_calibration_crystal"/>
    <addaction name="action_edit_apply_polar_mask"/>
    <addaction name="action_edit_reset_instrument_config"/>
    <addaction name="action_transform_detectors"/>
   </widget>
   <widget class="QMenu" name="menu_run">
    <property name="title">
     <string>R&amp;un</string>
    </property>
    <addaction name="action_run_powder_calibration"/>
    <addaction name="action_calibration_line_picker"/>
    <addaction name="action_run_indexing"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menuView"/>
   <addaction name="menu_run"/>
  </widget>
  <widget class="QStatusBar" name="status_bar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDockWidget" name="config_dock_widget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>312</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Configuration</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="config_dock_widgets">
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolBox" name="config_tool_box">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="materials_page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>550</width>
          <height>708</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Materials</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="load_page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>550</width>
          <height>708</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Load Data</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="instrument_calibration_page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>550</width>
          <height>708</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Instrument Calibration</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTabWidget" name="calibration_tab_widget">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="form_view">
            <attribute name="title">
             <string>Form View</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tree_view">
            <attribute name="title">
             <string>Tree View</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="color_map_dock_widget">
   <property name="windowTitle">
    <string>Color Map</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="color_map_dock_widgets">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="image_mode_dock_widget">
   <property name="windowTitle">
    <string>Image Mode</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="image_mode_dock_widgets">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <action name="action_tabbed_view">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Tab Images</string>
   </property>
   <property name="toolTip">
    <string>View images in tabs</string>
   </property>
  </action>
  <action name="action_open_images">
   <property name="text">
    <string>&amp;Images</string>
   </property>
  </action>
  <action name="action_open_config">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="action_save_imageseries">
   <property name="text">
    <string>&amp;ImageSeries</string>
   </property>
  </action>
  <action name="action_open_materials">
   <property name="text">
    <string>&amp;Materials</string>
   </property>
  </action>
  <action name="action_save_config">
   <property name="text">
    <string>&amp;Configuration</string>
   </property>
  </action>
  <action name="action_show_toolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Show Toolbar</string>
   </property>
  </action>
  <action name="actionShow_Live_Updates">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Live Updates</string>
   </property>
  </action>
  <action name="action_show_live_updates">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Live Updates</string>
   </property>
  </action>
  <action name="action_show_saturation_percentages">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Saturation &amp;Percentages</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Show the saturation percentages of the images when in &amp;quot;Image View&amp;quot; mode. The saturation percentages will appear as white text in the bottom left corners of the images.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="action_save_materials">
   <property name="text">
    <string>&amp;Materials</string>
   </property>
  </action>
  <action name="action_run_powder_calibration">
   <property name="text">
    <string>&amp;Powder Calibration</string>
   </property>
  </action>
  <action name="action_edit_euler_angle_convention">
   <property name="text">
    <string>Euler Angle &amp;Convention</string>
   </property>
  </action>
  <action name="action_open_aps_imageseries">
   <property name="text">
    <string>&amp;APS ImageSeries</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_edit_calibration_crystal">
   <property name="text">
    <string>Calibration Crystal</string>
   </property>
  </action>
  <action name="action_show_detector_borders">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Detector Borders</string>
   </property>
  </action>
  <action name="action_export_polar_plot">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Polar Plot</string>
   </property>
  </action>
  <action name="action_calibration_line_picker">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Calibration Line Picker</string>
   </property>
  </action>
  <action name="action_edit_apply_polar_mask">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Apply Mask to Polar</string>
   </property>
  </action>
  <action name="action_edit_reset_instrument_config">
   <property name="text">
    <string>Reset Instrument Config</string>
   </property>
  </action>
  <action name="action_open_template_dialog">
   <property name="text">
    <string>LLNL Images</string>
   </property>
  </action>
  <action name="action_transform_detectors">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Transform Detectors</string>
   </property>
  </action>
  <action name="action_run_indexing">
   <property name="text">
    <string>Indexing</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageTabWidget</class>
   <extends>QTabWidget</extends>
   <header>image_tab_widget.py</header>
   <container>1</container>
   <slots>
    <slot>open_files()</slot>
    <slot>set_tabbed_view(bool)</slot>
    <slot>show_nav_toolbar(bool)</slot>
    <slot>show_toolbar(bool)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_show_toolbar</sender>
   <signal>toggled(bool)</signal>
   <receiver>image_tab_widget</receiver>
   <slot>show_toolbar(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>1075</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>main_window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
